### ISSUE TITLE
Unable to remove policy from virtual server

### ISSUE THREAD

**Issue by chihaiaalex** [F5Networks/terraform-provider-bigip#383]
### Environment

 * TMOS/Bigip Version: BIG-IP 12.1.2 Build 2.0.276 Hotfix HF2
 * Terraform Version: v0.13.5
 * Terraform bigip provider Version: 1.4.0

### Summary

After attaching a policy to a virtual server, I can no longer detach it. Although the `terraform apply` command exists successfully,
the policy is not detached from the virtual server. If I do it manually from the UI, is working as expected.

### Steps To Reproduce

```
resource "bigip_ltm_pool" "test_pool" {
  name = "/k8s/pool_TCP31380_POOL"
  load_balancing_mode = "round-robin"
  description = "Test pool"
  monitors = [
    "/Common/tcp"
  ]
  allow_snat = "yes"
  allow_nat = "yes"
}

resource "bigip_ltm_virtual_server" "test_virtual_server_http" {
  name = "/k8s/fqdn_HTTP_VS"
  destination = "nnn.nn.nnn.nn"
  port = 80
  description = "Test virtual server"
  pool = bigip_ltm_pool.test_pool.name
  ip_protocol = "tcp"
  profiles = [
    "/Common/tcp",
    "/Common/http"
  ]
  persistence_profiles = [
    "/Common/cookie"
  ]
  source_address_translation = "automap"
  translate_address = "enabled"
  policies = [
    "test-policy"
  ]
}

resource "bigip_ltm_policy" "test_policy" {
  name = "/k8s/test-policy"
  strategy = "first-match"
  requires = ["http"]
  controls = [
    "persistence"
  ]

  rule {
    name = "enable-host-based-persistence"

    condition {
      http_host = true
      host = true
      equals = true
      values = [
        "hello.example.com"
      ]
    }
  }
  
  rule {
    name = "disable-persistence"

    action {
      disable = true
      persist = true
    }
  }
}

```
The output of running `terraform apply` on this code is the following

```
An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # module.bigip.bigip_ltm_virtual_server.test_virtual_server_http will be updated in-place
  ~ resource "bigip_ltm_virtual_server" "test_virtual_server_http" {
        client_profiles            = []
        description                = "Test virtual server"
        destination                = "nnn.nn.nnn.nn"
        id                         = "/k8s/fqdn_HTTP_VS"
        ip_protocol                = "tcp"
        irules                     = []
        mask                       = "255.255.255.255"
        name                       = "/k8s/fqdn_HTTP_VS"
        persistence_profiles       = [
            "/Common/cookie",
        ]
      ~ policies                   = [
          - "test-policy",
        ]
        pool                       = "/k8s/pool_TCP31380_POOL"
        port                       = 80
        profiles                   = [
            "/Common/http",
            "/Common/tcp",
        ]
        server_profiles            = []
        source                     = "0.0.0.0/0"
        source_address_translation = "automap"
        state                      = "enabled"
        translate_address          = "enabled"
        translate_port             = "enabled"
        vlans                      = []
        vlans_enabled              = false
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions in workspace "test"?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

module.bigip.bigip_ltm_virtual_server.test_virtual_server_http: Modifying... [id=/k8s/fqdn_HTTP_VS]
module.bigip.bigip_ltm_virtual_server.test_virtual_server_http: Modifications complete after 0s [id=/k8s/fqdn_HTTP_VS]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

```

### Expected Behavior

Although the result seems ok, if I go and check in the UI, the policy is still attached to the virtual server.

### Actual Behavior

The policy should be detached from the virtual server when running `terraform apply`.

**Comment by nmenant** [comment]
The engineering team will look into this, tracking internally with TER-513. Thanks for sharing this with us

**Comment by RavinderReddyF5** [comment]
@chihaiaalex Issue fixed through https://github.com/F5Networks/terraform-provider-bigip/pull/399
you can verify it on latest master branch clone.

**Comment by papineni87** [comment]
Issue Addressed in v1.6.0, please open new issues if you face any problem
